stevePObjects/src/carPackage/carClass.java

```diff
@@ -0,0 +1,41 @@
+package carPackage;
+
+/**
+ *Car class
+ * @author Steve
+ */
+public class carClass {
```

Comment:
I forgot that it is supposed to start with a capital, so I have fixed this problem.